pkgs/by-name/ze/zelda64recomp/package.nix

```diff
@@ -0,0 +1,173 @@
+{
+  lib,
+  llvmPackages,
+  llvm,
+  clang,
+  lld,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  pkg-config,
+  gtk3,
+  SDL2,
+  ares,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  mm-decomp,
+  n64recomp,
+  libXdmcp,
+  libxkbcommon,
+  libepoxy,
+  libXtst,
+  directx-shader-compiler,
+  makeWrapper,
+  vulkan-loader,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook3,
+}:
+
+let
+
+  # rt64 requires a specific version
+  directx' = directx-shader-compiler.overrideAttrs rec {
+    version = "1.8.2403.2";
+    src = fetchFromGitHub {
+      owner = "microsoft";
+      repo = "DirectXShaderCompiler";
+      rev = "v${version}";
+      hash = "sha256-cKvY67WWrMd6ALqgVmeDyRIXy7s6RAP8gRvB6BNKmEo=";
+      fetchSubmodules = true;
+    };
+  };
+in
+llvmPackages.stdenv.mkDerivation (finalAttrs: {
```

Comment:
Why llvm clang is needed?

Reply:
clang is a dependency in the upstream build instructions https://github.com/Zelda64Recomp/Zelda64Recomp/blob/v1.1.1/BUILDING.md